### DISCUSSION TITLE
How do I filter out tracing events?

### DISCUSSION THREAD

**Discussion by voroninp** [datalust/seq-tickets#2260] [General]
I want to see only log events and filter out traces with attached events.
Which property should I use to distinguish one from another?

**Comment by nblumhardt** [comment]
Hi @voroninp,

In "Signals" list, (perhaps by searching it), you'll find signals called "Logs" and "Spans", which should do what you're after.

In short though, `has(@Start)` will select only spans, and `not has(@Start)` will select only logs.

Hope this helps,
Nick

**Reply by voroninp** [reply]
Thanks. But the filtered stream still shows events from traces.
![image](https://github.com/user-attachments/assets/6e4eae43-0d96-49cd-88f3-37979648f602)

**Reply by nblumhardt** [reply]
Ah I see, thanks for the clarification. I don't think there's a specific method to do this currently, events embedded in traces are considered to be on an equal footing with other log events.
